PHPCI/Controller/BuildController.php

```diff
@@ -179,15 +179,49 @@ public function delete($buildId)
     }
 
     /**
-    * Parse log for unix colours and replace with HTML.
-    */
+     * Parse log for unix colours and replace with HTML.
+     *
+     * @link http://tldp.org/HOWTO/Bash-Prompt-HOWTO/x329.html
+     * @param string $log Output
+     */
     protected function cleanLog($log)
```

Comment:
Can you combine all the str_replace in one call? Now you have a str_replace with an array and after that two calls with one value. Or can you add comments if that is needed